src/Utils/Image.php

```diff
@@ -530,7 +530,7 @@ public function save($file = NULL, $quality = NULL, $type = NULL)
 				return imagegif($this->image, $file);
 
 			default:
-				throw new Nette\InvalidArgumentException('Unsupported image type \'$type\'.');
+				throw new Nette\InvalidArgumentException($type ? "Unsupported image type \'$type\'." : 'Unsupported file extension \'' . pathinfo($file, PATHINFO_EXTENSION) . '\'.');
```

Comment:
What about throwing about unsupported extension in the switch above in default branch (https://github.com/nette/utils/pull/72/files#diff-73701c083ecb416c68fb31139c302d43R530)?